Gems/ExternalProfilers/OptickProfiler/Code/CMakeLists.txt

```diff
@@ -0,0 +1,76 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+# Add Optick dependency using FetchContent
+include(FetchContent)
+FetchContent_Declare(
+    Optick
+    GIT_REPOSITORY "https://github.com/bombomby/optick"
```

Comment:
this works?  Isn't the actual clone address 
`https://github.com/bombomby/optick.git`

Reply:
Yes it was working but you're right better clean it up